ts-web-component/tsconfig.json

```diff
@@ -6,7 +6,7 @@
     "allowJs": false,
     "skipLibCheck": true,
     "esModuleInterop": false,
-    "allowSyntheticDefaultImports": true,
+    "allowSyntheticDefaultImports": false,
```

Comment:
Not sure if we have this as true in other places. It is a strange to allow this.